src/vs/workbench/contrib/userDataProfile/browser/userDataProfilesEditor.ts

```diff
@@ -1048,7 +1048,7 @@ class ProfileIconRenderer extends ProfilePropertyRenderer {
 			}, true);
 
 			if (hoverWidget) {
-				iconSelectBox.layout(new Dimension(486, 260));
+				iconSelectBox.layout(new Dimension(486, 292));
```

Comment:
Should we change here too?

https://github.com/microsoft/vscode/blob/a3b4487e6f5c57b9b49aa92e795e7367f0022cde/src/vs/workbench/contrib/terminal/browser/terminalIconPicker.ts#L54

Reply:
Profiles has relatively smaller set of icons and wanted them to fit the content.  In your case, I believe there is a large set you are not showing this as an hover like in profiles editor. So it should be fine not to have same height.